### PR TITLE
Changed brave-laptop to brave-browser.

### LINKED ISSUE #325: Brave-Laptop repository currently linked is now read-only
The `brave-laptop` repository currently linked is now read only and they have shifted their code base to a new repo namely, `brave-browser`.

Seems like a simple fix, would love to work on this and submit a PR

cc: @MunGell

### PR DESCRIPTION
**Pull request by co16353sidak**
Changed the `brave-laptop` reference (as the repo is now read-only) and changed it into `brave-browser` which now contains the current code-base.
Fixes #325

### CODE REVIEW

**Comment by co16353sidak** [comment]
`brave-browser` does no longer have a first issue or first bug label anymore, their documentation issues are a good place to start work and I have added that label in the link.

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [stryker](https://github.com/stryker-mutator/stryker/labels/beginner%20friendly) _(label: beginner friendly)_ <br> The JavaScript mutation testing framework
 - [Kap](https://github.com/wulkano/kap/labels/good%20first%20issue) _(label: good first issue)_ <br> An open-source screen recorder built with web technology
 - [Reddit Enhancement Suite](https://github.com/honestbleeps/Reddit-Enhancement-Suite/labels/help%20wanted) _(label: help wanted)_ <br> A browser extension to enhance the Reddit browsing experience.
-- [Brave Browser](https://github.com/brave/browser-laptop/labels/bug%2Fgood-first-bug) _(label: bug/good-first-bug)_ <br> Desktop browser for macOS, Windows, and Linux.
+- [Brave Browser](https://github.com/brave/brave-browser/labels/documentation%20%3Awriting_hand%3A) _(label: documentation)_ <br> Desktop browser for macOS, Windows, and Linux.
```
Comment (co16353sidak) [inline README.md]: the documentation label is linked and that label is also mentioned in brackets.

Comment (vvscode) [reply README.md]: You have `is:open label:"documentation :writing_hand:" ` in link
and `_(label: documentation)_` in brackets

Comment (vvscode) [reply README.md]: while actual label is `documentation` - looks like just update the link

Comment (co16353sidak) [reply README.md]: `is:open` is just simply to filter out only those issues which are currently open on the repo.

Comment (vvscode) [reply README.md]: you have not `is: open` , you have `:writing_hand:`

**Comment by MunGell** [comment]
@co16353sidak why did you change the label to be documentation? Why do you think it is the one this repository owners use for beginners friendly issues and not something like https://github.com/brave/brave-browser/labels/good%20first%20bug ?

**Comment by co16353sidak** [comment]
Hey @MunGell 
brave-browser doesn't seem to have a tag like `good first bug` or `good first issue` but on going through the issues I have found that the issues tagged with `documentation` seem to have beginner friendly issues

**Comment by MunGell** [comment]
Well, in this case we should either just remove brave from the list or contact maintainers so they could clarify what tag they use (if any) for beginner-friendly issue.

**Comment by co16353sidak** [comment]
@MunGell  I don't think they now identify as a beginner friendly repo anymore. If you agree then I will remove the entry from the list.

**Comment by MunGell** [comment]
Hi @bbondy, can you help us with the issue above?

**Comment by bbondy** [comment]
Hi guys, thanks for the ping. I think you're looking for:
https://github.com/brave/brave-browser/labels/good%20first%20bug

**Comment by MunGell** [comment]
Thanks @bbondy 

@co16353sidak would you mind updating this PR?

**Comment by co16353sidak** [comment]
@MunGell making the changes now.

**Comment by MunGell** [comment]
Thank you

**Pull request merged**